android/keyple-plugin/android-omapi/src/main/kotlin/org/eclipse/keyple/plugin/android/omapi/simalliance/AndroidOmapiReader.kt

```diff
@@ -64,25 +62,23 @@ internal class AndroidOmapiReader(private val nativeReader: Reader, pluginName:
      * Open a logical channel by selecting the application
      * @param aidSelector the selection parameters
      * @return a ApduResponse built from the FCI data resulting from the application selection
-     * @throws KeypleIOReaderException
-     * @throws KeypleChannelControlException
-     * @throws KeypleApplicationSelectionException
+     * @throws KeypleReaderIOException if the communication with the reader or the SE has failed
      */
-    @Throws(KeypleIOReaderException::class, KeypleChannelControlException::class, KeypleApplicationSelectionException::class)
```

Comment:
Why did you reduce the ability of our users to implement a fine management of this exceptions?